processing/data_collection/gazette/pipelines.py

```diff
@@ -141,3 +155,11 @@ def _is_file_type(self, filepath, file_types):
         Generic method to check if a identified file type matches a given list of types
         """
         return self.get_file_type(filepath) in file_types
+
+
+class GazetteFilesPipeline(FilesPipeline):
+    def get_media_requests(self, item, info):
+        return [
+            x if isinstance(x, Request) else Request(x)
+            for x in item.get(self.files_urls_field, [])
+        ]
```

Comment:
I think we can remove this as well. No images anymore

Reply:
Done.

This was not implemented due to the files being images. It was because the system required cookies and Scrapy's FilesPipeline doesn't provide a way of downloading files with this constraint yet.